wgpu-core/src/device/queue.rs

```diff
@@ -1162,8 +1162,8 @@ impl Global {
             let snatch_guard = device.snatchable_lock.read();
 
             // Fence lock must be acquired after the snatch lock everywhere to avoid deadlocks.
-            let mut fence = device.fence.write();
-            let fence = fence.as_mut().unwrap();
+            let mut fence_guard = device.fence.write();
+            let fence = fence_guard.as_mut().unwrap();
```

Comment:
thought: It woulda been nice to have mechanical renames in a separate commit, but no point in changing that now that I understand what's going on. 🫡